### PR TITLE
Feature: added routing for my-archive tabs.

### PR DESCRIPTION
**Pull request by benjaminnaesen**

### CODE REVIEW

File: src/my-archive/routes.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Bookmarks } from './views/Bookmarks';
+import { Collections } from './views/Collections';
+import { Favourites } from './views/Favourites';
+import { Folders } from './views/Folders';
+
+export const MY_ARCHIVE_ROUTES = [
+	{
+		path: '/my-archive/collections',
```
Comment (bertyhell) [inline src/my-archive/routes.ts]: routes have to be in dutch from now on (decision by VIAA/Bart)
**mijn archief** will be called: **mijn werkruimte** (https://district01.atlassian.net/browse/AVO2-363)
lets make the internal variables: `my-workspace`

File: src/my-archive/routes.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	},
+	{
+		path: '/my-archive/bookmarks',
+		exact: true,
+		component: Bookmarks,
+	},
+	{
+		path: '/my-archive/favourites',
+		exact: true,
+		component: Favourites,
```
Comment (bertyhell) [inline src/my-archive/routes.ts]: favorites and bookmarks will be merged into one: Bookmarks

File: src/my-archive/routes.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export const MY_ARCHIVE_ROUTES = [
+	{
+		path: '/my-archive/collections',
+		exact: true,
+		component: Collections,
+	},
+	{
+		path: '/my-archive/folder',
+		exact: true,
+		component: Folders,
```
Comment (bertyhell) [inline src/my-archive/routes.ts]: Collections and folders will be merged into one: Collections

File: src/my-workspace/views/Bookmarks.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React, { Fragment, FunctionComponent } from 'react';
+
+import {} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+import { RouteComponentProps } from 'react-router';
+
+interface BookmarksProps extends RouteComponentProps {}
+
+export const Bookmarks: FunctionComponent<BookmarksProps> = ({  }: BookmarksProps) => {
```
Comment (andrych) [inline src/my-workspace/views/Bookmarks.tsx]: Je moet die `: BookmarkProps` niet meer specifiek definiëren in uw function params aangezien dat hier `: FunctionComponent<BookmarksProps>` al gebeurd.

File: src/my-workspace/views/Collections.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React, { Fragment, FunctionComponent } from 'react';
+
+import {} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+import { RouteComponentProps } from 'react-router';
+
+interface CollectionsProps extends RouteComponentProps {}
+
+export const Collections: FunctionComponent<CollectionsProps> = ({  }: CollectionsProps) => {
```
Comment (andrych) [inline src/my-workspace/views/Collections.tsx]: idem comment hierboven

File: src/routes.ts [hunk 6]
```diff
@@ -1,5 +1,11 @@
 import { COLLECTION_ROUTES } from './collection/routes';
 import { ITEM_ROUTES } from './item/routes';
+import { MY_WORKSPACE_ROUTES } from './my-workspace/routes';
 import { SEARCH_ROUTES } from './search/routes';
 
-export const ROUTES: any[] = [...SEARCH_ROUTES, ...ITEM_ROUTES, ...COLLECTION_ROUTES];
```
Comment (andrych) [inline src/routes.ts]: `RouteProps` van react-router types gebruiken ipv `any`.

**Pull request merged**